src/frontend.c

```diff
@@ -111,7 +140,18 @@ int main(int argc, char *argv[]) {
   };
   State *state = &new_state;
 
-  while ((state->ch_in = wgetch(canvas_win))) {
+  while (1) {
```

Comment:
Rather than make a "file read/write" mode, I added another layer that intercepts keypresses before they're sent to the modes, which lets you use these commands in any mode.

I'm not sure about the function name or the method of passing in the window vs a read character. Thoughts?

Reply:
No longer relevant - overhauled in #45